datamover/pkg/drivers/lifecycle/incloud_transition.go

```diff
@@ -51,7 +51,10 @@ func doInCloudTransition(acReq *datamover.LifecycleActionRequest) error {
 		return errors.New(DMERR_TransitionInprogress)
 	}
 
-	ctx, _ := context.WithTimeout(context.Background(), CLOUD_OPR_TIMEOUT*time.Second)
+	// min is 1 minute, max is 30 days, default is 1 hour, user defined value should not less than min and not more than
+	// max, otherwise default will be used
+	tmout := migration.GetCtxTimeout("OBJECT_MOVE_TIME", SECONDS_ONE_MINUTE, SECONDS_30_DAYS, SECONDS_ONE_HOUR)
+	ctx, _ := context.WithTimeout(context.Background(), tmout)
```

Comment:
question here, say the timeout expires, all files are NOT yet moved...is there a persisted log someplace or any events on UI, to show this to the user, to retry the incomplete items ?

Reply:
Hi @rhsakarpos There are two scenarios related, lifecycle management and migration. For lifecycle management, if it failed, we just print an warn log and it will try again in the next round, lifecycle scheduler runs periodically. For migration, the job will return and set the status to be failed.

Reply:
sounds good. However, will migration also list which objects were migrated and which were not ?

Reply:
No, not list now, it's only print in log, maybe we can add this part in the future in some way, cause objects the number of objects maybe huge.

Reply:
sure, log is also good, something for the user to see and act on